src/uu/join/src/join.rs

```diff
@@ -444,9 +445,11 @@ impl<'a> State<'a> {
 
         if diff == Ordering::Greater {
             eprintln!(
-                "{}:{}: is not sorted",
+                "{}: {}:{}: is not sorted: {}",
+                uucore::util_name(),
```

Comment:
For warnings and errors we prefer `uucore::execution_phrase()` over `uucore::util_name()`. The util name is static while the execution phrase depends on how it was called. You can see this with GNU join as well:
```
❯ join fields_2.txt fields_4.txt
2 b c 1 cd
3 c d 2 de
5 e e 3 ef
join: fields_4.txt:5: is not sorted: 11 g 5 gh
7 g f 4 fg
join: input is not in sorted order
```
```
❯ /usr/bin/join fields_2.txt fields_4.txt                                                                                                                               11:29:09
2 b c 1 cd
3 c d 2 de
5 e e 3 ef
/usr/bin/join: fields_4.txt:5: is not sorted: 11 g 5 gh
7 g f 4 fg
/usr/bin/join: input is not in sorted order
```